packages/rest/test/integration/rest.server.integration.ts

```diff
@@ -246,20 +242,148 @@ servers:
       '/swagger-ui',
     );
     await server.get(RestBindings.PORT);
-    const url = new RegExp(
+    const urlString = new RegExp(
```

Comment:
Ditto: the value is a regular expression, not a string. Please use `urlMatcher` or perhaps `expectedUrl` instead.